eventlet/__init__.py

```diff
@@ -21,8 +21,12 @@
     # Helpful when CPython < 3.5 on Linux blocked in `os.waitpid(-1)` before first use of hub.
     # Example: gunicorn
     # https://github.com/eventlet/eventlet/issues/401#issuecomment-327500352
-    import monotonic
-    del monotonic
+    try:
+        import monotonic
+        del monotonic
+    except ImportError:
+        import time
+        del time
```

Comment:
```suggestion
        pass
```
`import time` is not useful, right?